src/plugins/intel_cpu/tests/functional/single_layer_tests/matmul.cpp

```diff
@@ -758,7 +782,7 @@ const auto fullyConnectedParams2D_Brgemm_nightly = ::testing::Combine(::testing:
 
 const auto testParams2D_Brgemm_nightly = ::testing::Combine(fullyConnectedParams2D_Brgemm_nightly,
                                              ::testing::Values(MatMulNodeType::FullyConnected),
-                                             ::testing::ValuesIn(fusingParamsSet2D_nightly),
+                                             ::testing::ValuesIn(filterFusingParams(fusingParamsSet2D_nightly)),
```

Comment:
Have you tried this `nightly` test locally? From my understanding it should pass since it expects brgemm impl to be chosen, but actually mlas will be launched. Is my understadning correct?

Reply:
Yes I have try the nightly test locally, I have separate that MLAS tests from original tests.